src/main/kotlin/model/SnowballRException.kt

```diff
@@ -58,4 +61,41 @@ sealed class SnowballRException(
             criterionId: String,
         ) : EntityNotPersistedException("Criterion", criterionId)
     }
+
+    /**
+     * Represents a specific type of exception that occurs when an entity is accessed by
+     * the current user, but they are not authorized.
+     *
+     * This exception is intended to provide a clear and structured way to handle
+     * scenarios where a particular entity is accessed, but the current user is not
+     * authorized to do so.
```

Comment:
```suggestion
     * scenarios where a particular entity is accessed without permission.
```
Otherwise you only repeat yourself unnecessarily?